torchgeo/trainers/segmentation.py

```diff
@@ -36,19 +36,30 @@ def config_task(self) -> None:
         """Configures the task based on kwargs parameters passed to the constructor."""
         weights = self.hyperparams["weights"]
 
+        if self.hyperparams["backbone"].startswith("tu-vit"):
+            encoder_depth = 4
+            decoder_channels = (256, 128, 64, 32)
```

Comment:
```suggestion
            decoder_channels: tuple[int, ...] = (256, 128, 64, 32)
```
This should placate mypy